src/components/app/index.tsx

```diff
@@ -17,6 +18,11 @@ const ENCODER_NAMES = {
   MozJpeg: 'JPEG'
 };
 
+const ENCODER_EXTENSION = {
+  original: '',
+  MozJpeg: 'jpeg'
```

Comment:
Anyone else twitching at this? I'd go for `jpg`, but I don't mind if it's just me.

Also, it feels like this is information that should be owned by something more related to the encoder, not at the top level. I'm happy for this to merge as long as we create an issue to refactor this.

Reply:
stop living in DOS-land man! :)